action.yml

```diff
@@ -1,35 +1,134 @@
 name: Setup Nox
-description: 'Prepares all python versions for nox'
+description: "Prepares all python versions for nox"
+inputs:
+  python-versions:
+    description: "comma-separated list of python versions to install"
+    required: true
+    default: "3.7, 3.8, 3.9, 3.10, pypy-3.7, pypy-3.8, pypy-3.9"
```

Comment:
Should this be `required: false` if we're specifying a default? i.e. would a user _have_ to specify this or could they just do something like:

```yaml
- name: Setup Nox
  uses: wntrblm/nox
```

Reply:
done